Installation/cmake/modules/FindGMP.cmake

```diff
@@ -31,15 +31,33 @@ if( NOT GMP_in_cache )
   	        DOC "The directory containing the GMP header files"
            )
 
-  find_library(GMP_LIBRARIES NAMES gmp libgmp-10 mpir
+  find_library(GMP_LIBRARY_RELEASE NAMES gmp libgmp-10 mpir
     HINTS ENV GMP_LIB_DIR
           ENV GMP_DIR
           ${CGAL_INSTALLATION_PACKAGE_DIR}/auxiliary/gmp/lib
     PATH_SUFFIXES lib
-    DOC "Path to the GMP library"
+    DOC "Path to the Release GMP library"
     )
 
-  if ( GMP_LIBRARIES )
+  find_library(GMP_LIBRARY_DEBUG NAMES gmpd gmp libgmp-10 mpir
+    HINTS ENV GMP_LIB_DIR
+          ENV GMP_DIR
+          ${CGAL_INSTALLATION_PACKAGE_DIR}/auxiliary/gmp/lib
+    PATH_SUFFIXES lib
+    DOC "Path to the Debug GMP library"
+    )
+
+  get_property(IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+  if(IS_MULTI_CONFIG)
+  set(GMP_LIBRARIES debug ${GMP_LIBRARY_DEBUG} optimized ${GMP_LIBRARY_RELEASE})
+else()
+  if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+    set(GMP_LIBRARIES ${GMP_LIBRARY_DEBUG})
+  else()
+    set(GMP_LIBRARIES ${GMP_LIBRARY_RELEASE})
+  endif()
+endif()
+   if ( GMP_LIBRARIES )
```

Comment:
The indentation pattern is not right.

And the line 61 (https://github.com/CGAL/cgal/pull/5578/files#diff-8ea666e9d7c8d634a294f5f2747c2515644ead1c9154cbed00654b8b8030b1e7R61) cannot run correctly. You should remove that variable `GMP_LIBRARIES_DIR` from `FindGMP.cmake`.